docs/api-types/tls-route.md

```diff
@@ -0,0 +1,58 @@
+# TLSRoute API Reference
+
+## Introduction
+
+With integration of the Gateway API, AWS Gateway API Controller supports `TLSRoute`.
+This allows you to define and manage end-to-end TLS encrypted traffic routing to your Kubernetes clusters.
+
+### Considerations
+
+- `TLSRoute` sectionName must refer to an `TLS` protocol listener with `mode: Passthrough` in the parentRefs `Gateway`.
+- `TLSRoute` only supports to have one rule.
```

Comment:
perhaps to be more specific, only supports one default rule

Reply:
But I think Kubernetes resource TLSRoute don't have a concept of "default rule" ?
https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1alpha2.TLSRoute

Reply:
ok, i misunderstood. or the wording can be "can only have one rule"

Reply:
I did consider to add "Features" and "Limitations"， but don't think TLSRoute has some fancy features that need one separate section besides some words in the "Introduction" and " Example Configuration" sections.

For "Limitations", just use the word "Considerations", same as the word in the [vpc lattice doc](https://docs.aws.amazon.com/vpc-lattice/latest/ug/tls-listeners.html#tls-listeners-considerations) because that are more likely design decision and not limitation? What do you think?

Reply:
OK I think it's fine to leave as-is